src/main/java/com/google/sps/servlets/DisplayUnrepliedPostsServlet.java

```diff
@@ -0,0 +1,53 @@
+//need add verification of owner
+
+package com.google.sps.servlets;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.cloud.datastore.Key;
+import com.google.cloud.datastore.KeyFactory;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for displaying posts. */
+@WebServlet("/display-posts-unreplied")
+public class DisplayUnrepliedPostsServlet extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
```

Comment:
Can you add a "TODO" comment here saying we should unify the display post endpoint with Mia's work (i.e. removing one of the files and completing the datastore entity so that we only have one file and endpoint for displaying (unreplied) posts).